hw/lint/tools/veriblelint/parse-lint-report.py

```diff
@@ -82,10 +95,13 @@ def main():
                         files in this directory.
                         Defaults to './'""")
 
-    parser.add_argument('--outdir',
-                        type=str,
-                        default="./",
-                        help="Output directory for the 'results.hjson' file. Defaults to '%(default)s'")
+    parser.add_argument(
+        '--outdir',
+        type=str,
+        default="./",
+        help=
+        "Output directory for the 'results.hjson' file. Defaults to '%(default)s'"
+    )
```

Comment:
Pleaes do not add these diffs to the patch. They are known to be unstable in yapf.

Reply:
ok, I reverted this part and broke the long line by hand.